docs/INSTALL.md

```diff
@@ -201,6 +230,16 @@ ln -s ../data/ data
 
 Some tests [are known to be broken](https://github.com/organicmaps/organicmaps/issues?q=is%3Aissue+is%3Aopen+label%3ATests).
 
+### Changing themes
```

Comment:
Should it be in contributing docs? Maybe add a link from here to there?

Reply:
This INSTALL.md file is the main place that describes desktop app so I think this "hidden" options should be documented here.

But I will change this to text to focus more on that these theme command are not only for theme changing